components/mailbox/src/Mailbox.svelte

```diff
@@ -542,9 +542,9 @@
       }
 
       &:hover {
-        $hover-outline-width: 1px;
-        border: $hover-outline-width solid var(--grey-warm);
+        $hover-outline-width: 0.5px;
         border-top-width: 0;
+        border: $hover-outline-width solid var(--grey-warm);
```

Comment:
Can this be changed to `outline` so the UI doesn't shift when the `li` has a hover state?